### PR TITLE
multiple pages : Add Korean Translation

### PR DESCRIPTION
**Pull request by Proscream**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by sbrl** [comment]
I'm confused. Which PR is which, @Proscream? I see the following PRs:

 - #3560
 - #3556
 - #3555
 - #3553

**Comment by Proscream** [comment]
@sbrl I don't know whats mean. Do I need to change?

**Comment by Proscream** [comment]
@einverne thanks for a comment. I'll do that.

File: pages.ko/common/7zr.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # 7zr
 
-> 높은 파일 압축률을 보여주는 압축 프로그램.
-> .7z파일들만을 지원하는 `7z`의 독립형 버전.
+> 높은 파일 압축률을 보여주는 압축 프로그램
+> .7z파일들만을 지원하는 `7z`의 독립형 버전
```
[the remark this answers is not in the document]
Comment (Proscream) [reply pages.ko/common/7zr.md]: I'm confuesd about that.  In pages/common/7zr.md,
"A file archiver with high compression ratio. A standalone version of 7z that only supports .7z files. More information: https://www.7-zip.org/."
So I used a period.

File: pages.ko/common/apropos.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# apropos
+
+> 메뉴얼 페이지에서 검색, 예를 들어 새로운 명령어 검색.
+
+- 키워드 검색:
+
+`apropos {{regular_expression}}`
```
Comment (sbrl) [inline pages.ko/common/apropos.md]: Perhaps `regular_expression` could be translated here too? Same for the rest of pages.

Also, don't worry about the comment above. It was me getting confused at your multiple PRs containing multiple translated pages, and being unable to tell the difference between them :P

Comment (Proscream) [reply pages.ko/common/apropos.md]: Of course i can do that. And thank you. In the future, when I leave many PR, I will change title a little.

File: pages.ko/common/7zr.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Proscream) [reply pages.ko/common/7zr.md]: And in #3560, you recommend use period. How can i do?

[the remark this answers is not in the document]
Comment (einverne) [reply pages.ko/common/7zr.md]: In this markdown, period is needed.

File: pages.ko/common/7zr.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # 7zr
 
-> 높은 파일 압축률을 보여주는 압축 프로그램.
-> .7z파일들만을 지원하는 `7z`의 독립형 버전.
+> 높은 파일 압축률을 보여주는 압축 프로그램
```
Comment (einverne) [inline pages.ko/common/7zr.md]:
```suggestion
> 높은 파일 압축률을 보여주는 압축 프로그램.
```

File: pages.ko/common/7zr.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Proscream) [reply pages.ko/common/7zr.md]: If so, you removed the period of 7zr, how do we distinguish between what to keep and what to delete?

[the remark this answers is not in the document]
Comment (agnivade) [reply pages.ko/common/7zr.md]: @Proscream A period is always needed at the end of a heading. Can you clarify where is the confusion ? Did somebody ask you to remove the period ? From what I see, @einverne has asked you to add the period.

[the remark this answers is not in the document]
Comment (Proscream) [reply pages.ko/common/7zr.md]: @agnivade Looks like he deleted the period in this comment, isn't it? It seems to me that deleted the sentence with a period and added a sentence without the period. He asked me to keep the period but deleted it in the comment. So it was confusing to me.
Then, should I keep the period, right?

[the remark this answers is not in the document]
Comment (agnivade) [reply pages.ko/common/7zr.md]: Oh, yea I believe you are right. His comment and actions are opposite. Yes, you should keep the period. Thanks.

[the remark this answers is not in the document]
Comment (Proscream) [reply pages.ko/common/7zr.md]: @agnivade Thank you for your comments. Now I can PR again!

**Comment by Proscream** [comment]
it induce conflict because it is duplicated. @einverne

**Comment by einverne** [comment]
If this PR is merged, tldr page will be deleted. In this PR, you do not need to delete tldr page, I only found one page under `pages.ko/common/tldr.md`, if you do not create new one there is no duplication.

**Comment by Proscream** [comment]
Thanks! I will re-commit tldr.md

**Pull request merged**